assets/src/components/amp-support/errors.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * To render error information on site support page.
+ *
+ * @param {Object} props      Component props.
+ * @param {Object} props.data Error data.
+ * @return {JSX.Element|null} HTML markup for error data.
+ */
+export function Errors( { data: errors } ) {
+	if ( 'object' !== typeof errors ) {
+		return null;
+	}
+
+	return (
+		<details open={ false }>
+			<summary>
+				{ __( 'Errors', 'amp' ) }
+				{ ` (${ errors.length || 0 })` }
```

Comment:
An empty array's length equals 0 so there's no need for the OR operator. Also, the entire title should be localizable, i.e.:

```suggestion
				{ sprintf(
					/* translators: Placeholder is the number of errors */
					__( 'Errors (%d)', 'amp' ),
					data.errors.length,
				) }
```